### ISSUE TITLE
Mailchimp for WP on GAE : no menu item

### ISSUE THREAD

**Issue by CristKa** [dannyvankooten/mailchimp-for-wordpress#88]
When I am not using any other plugin with a menu item on the dashboard, the Mailchimp for WP menu item does not appear, although the plugin looks activated and has a working API key.

Platform : GAE, local and remote, WP 4.1.1, MailChimp for WordPress Lite 2.2.8

**Comment by dannyvankooten** [comment]
Hi @CristKa ,

Thanks for the bug report, will look into it! Can you please elaborate on what "GAE" stands for? Might be obvious but I'm missing it. :smile:

**Comment by CristKa** [comment]
Hi @dannyvankooten, thanks for your reply ! 

By GAE i meant Google App Engine, I should have detailed more.  The missing menu item happens when the engine runs locally, or remotely on Google server.

I used the default installation procedure :
 
https://googlecloudplatform.github.io/appengine-php-wordpress-starter-project/

As for now, I solved it by installing an other plugin with a menu, it made the MC4WP menu appear.
